src/dotnet/commands/dotnet-help/LocalizableStrings.resx

```diff
@@ -265,7 +265,7 @@
     <value>Version of the installed Shared Framework to use to run the application.</value>
   </data>
   <data name="RollForwardOnNoCandidateFxDefinition" xml:space="preserve">
-    <value>Roll forward on no candidate shared framework is enabled.</value>
+    <value>Set roll forward on no candidate shared framework behavior.</value>
```

Comment:
We need to get the values in here. Trying to shorten to fit.  

`<value>Roll forward on no candidate framework (0=off, 1=roll minor & patch, 2=roll major, minor & patch)</value>`

I do not like the word Set as I think it implies the setting holds beyond the current command which I believe it does not

Reply:
That sounds good. I would leave out the `patch` completely. It's a bit confusing, since `0=off` will actually by default roll forward patches as well, but it's goverened by a different setting.
In general we typically state that we roll forward patch version always.